src/main/java/com/zendesk/maxwell/schema/ddl/MysqlParserListener.java

```diff
@@ -367,12 +367,23 @@ public void exitColumn_definition(mysqlParser.Column_definitionContext ctx) {
 		}
 
 		colType = ColumnDef.unalias_type(colType.toLowerCase(), longStringFlag, columnLength, byteFlagToStringColumn);
-		ColumnDef c = ColumnDef.build(name,
-					                   colCharset,
-					                   colType.toLowerCase(),
-					                   -1,
-					                   signed,
-					                   enumValues);
+    ColumnDef c;
+    if (columnLength != null) {
+      c = ColumnDef.build(name,
+  														 colCharset,
```

Comment:
oh I see you were following my lame-ass whitespace.  sorry, the above is the new whitespace method